lib/exit.js

```diff
@@ -69,4 +77,73 @@ export default class Exit {
     buf.copy(txData, 0, offset, offset + length);
     return txData;
   }
+
+  static fastSellUTXO(utxo, plasmaChain, rootChain, marketMakerUrl, signer) {
+    return plasmaChain.getConfig().then(nodeConfig => {
+      const inputs = [new Input(utxo.outpoint)];
+      const outputs = [
+        new Output(utxo.output.value, nodeConfig.exitHandlerAddr, utxo.output.color)
+      ];
+      const exitingUtxoTransfer = Tx.transfer(inputs, outputs);
+      return Exit.fastSell(exitingUtxoTransfer, plasmaChain, rootChain, marketMakerUrl, signer);
+    })
+  }
+
+  static fastSellAmount(account, amount, color, plasmaChain, rootChain, marketMakerUrl, signer) {
```

Comment:
usage example for `signer` param: https://github.com/leapdao/burner-wallet/pull/24/files#diff-b00b060794e8d5797bffd335bd17e2a1R1435

Reply:
Btw, this param is missing in d.ts file